nvidia-nano/GiveMeWES.yaml

```diff
@@ -0,0 +1,597 @@
+---
+- name: Check Jetpack version
+  hosts: all
```

Comment:
update the `hosts` to be something more specific that is in the inventory file (see how NVidia did it)

Reply:
will do in a later update